packages/eas-cli/src/utils/git.ts

```diff
@@ -38,7 +43,15 @@ async function doesGitRepoExistAsync(): Promise<boolean> {
   }
 }
 
-async function isGitInstalledAsync(): Promise<boolean> {
+export async function gitCommitHashAsync(): Promise<string | undefined> {
+  try {
+    return await (await spawnAsync('git', ['rev-parse', 'HEAD'])).stdout.trim();
+  } catch (err) {
+    return undefined;
```

Comment:
When this could happen?

Reply:
Short answer: I don't know. 
Long answer: Slack